pytorch_lightning/trainer/trainer.py

```diff
@@ -2786,6 +2788,17 @@ def configure_optimizers(self):
         return max_estimated_steps
 
 
+@contextmanager
+def _evaluation_context() -> Generator:
+    cm = (
```

Comment:
nit: we can spare some letters :)

```suggestion
    context_manager_class = (
```

Reply:
done!